### PR TITLE
refactor: migrate template in plugins to i18n

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: packages/cli/src/worker/daemon.ts [hunk 1]
```diff
@@ -32,21 +26,27 @@ export function apply(ctx: Context, config: Config = {}) {
...
-      process.exit(51)
-    })
+  if (exitCommand) {
+    const commandName = exitCommand === true ? 'exit' : exitCommand
+
+    ctx.i18n.define('zh', { [`commands.${commandName}`]: require('../locales/zh') })
+
+    ctx
+      .command(commandName, { authority: 4 })
+      .option('restart', '-r  重新启动')
```
Comment (Shigma) [inline packages/cli/src/worker/daemon.ts:36]: 这一行后面中文不需要了

File: packages/cli/src/worker/daemon.ts [hunk 2]
```diff
@@ -32,21 +26,27 @@ export function apply(ctx: Context, config: Config = {}) {
...
+      .shortcut('关机', { prefix: true })
+      .shortcut('重启', { prefix: true, options: { restart: true } })
```
Comment (Shigma) [inline packages/cli/src/worker/daemon.ts:38]: 这两行删了算了，让用户自己加（反正文档也没写

**Comment by MaikoTan** [comment]
plugin-chat not tested currently, may test it at night or somewhen. Others looks fine.

**Comment by MaikoTan** [comment]
By the way, why is the CI not functionate at all.

**Comment by MaikoTan** [comment]
> plugin-chat not tested currently, may test it at night or somewhen. Others looks fine. 

Looks like the plugin-chat is okay, I think it's good to merge.

**Review by Shigma — APPROVED** [review]

**Comment by Shigma** [comment]
This will partially resolve #613.

**Pull request merged**